### PR TITLE
Additional OSS information

### PR DESCRIPTION
**Pull request by JulianRoesner**
We were discussing whether we should include information from [the open source template](https://github.com/digitalservicebund/opensource-template/tree/main) in our other templates at what information.

So far, we include the code of conduct [in the typescript template](https://github.com/digitalservicebund/typescript-vite-application-template) and the security policy in [the java template](https://github.com/digitalservicebund/java-application-template). Both have no contribution section in the readme.

From the discussions I had so far, I see the following points:
- We miss a lot of the information from the open source template in our repositories (e.g. contribution section, code of conduct). Having this in our templates could help us to spread information about it across our projects.
- The templates are also open source and should include information like the security policy & the CoC, too.
- This increases the maintenance overhead. The project templates and the open source template would not be separate and teams can combine them when starting a new project. We would need to maintain the files in the open source template and the projects templates.

Still, as all templates are open source I would propose the following:
- The open source template is the single source of truth and the point to add updates etc..
- The project templates include the information from the open source template as a normal project would. This means they do not give guidance on those files.

This PR should include the necessary changes.

What do you think?

**TLDR:** What files from the open source template should we include in the project templates?

### CODE REVIEW

File: SECURITY.md
Comment (kaibernhard) [inline SECURITY.md:1]: We can refer to this security policy in security.txt like we do in the Kotlin template, see https://github.com/digitalservicebund/kotlin-application-template/blob/main/src/main/resources/public/.well-known/security.txt#L4
`Policy: https://raw.githubusercontent.com/digitalservicebund/remix-application-template/main/SECURITY.md`

**Review by chrkaatz — APPROVED** [review]
💯

File: README.md [hunk 1]
```diff
@@ -114,3 +114,32 @@ For adding new records the [adr-tools](https://github.com/npryce/adr-tools) comm
 ```bash
 brew install adr-tools
 ```
+
+## Contributing
+
+🇬🇧
+Everyone is welcome to contribute the development of the _<please_exchange_with_project_name>_. You can contribute by opening pull request,
```
Comment (kaibernhard) [inline README.md]: Replace `_<please_exchange_with_project_name>_` with `remix-application-template`?
`run.sh init` should automatically do the replacements to minimise manual work when applying the template.

Comment (JulianRoesner) [reply README.md]: Yes, thanks. I forgot to push this change.

**Review by mpanne — APPROVED** [review]
The fact that the template itself is open source and should include all those information as well convinced me. 

Thanks for the effort!
